manifests/4.6/sriov-network-operator-sriovnetworknodestate.crd.yaml

```diff
@@ -57,15 +57,10 @@ spec:
                           type: string
                         vfRange:
                           type: string
-                      required:
-                      - policyName
```

Comment:
Are we allowing empty `policyName` and `resourceName`? if resource name is empty, how will a pod request the resource?

Reply:
SriovNetworkNodeState CRs are managed by the operator itself. So we shall not worry about the illegal configuration too much.